src/includes/getting-started-config/android.timber.mdx

```diff
@@ -28,6 +16,32 @@ SentryAndroid.init(this) { options ->
         minBreadcrumbLevel = SentryLevel.INFO
       )
     )
+  } else {
+    Timber.plant(Timber.DebugTree())
   }
 }
 ```
+
+```java
+import io.sentry.android.core.SentryAndroid;
+import io.sentry.android.timber.SentryTimberIntegration;
+import timber.log.Timber
+// import BuildConfig
+
+SentryAndroid.init(this, options -> {
+   if (!BuildConfig.DEBUG) {
```

Comment:
Just for me to understand: Why only when not debug? Wouldn't it be nice to see the events in Sentry?

Reply:
then you have `Timber.DebugTree`, some people don't want to report to Sentry debug levels, as they can see in the console, otherwise they pay for what they don't need to.